pontoon/administration/forms.py

```diff
@@ -25,6 +25,9 @@ class ProjectForm(forms.ModelForm):
     locales_readonly = forms.ModelMultipleChoiceField(
         queryset=Locale.objects.all(), required=False,
     )
+    locales = forms.ModelMultipleChoiceField(
+        queryset=Locale.objects.all(), required=False,
+    )
```

Comment:
Good work!

We should still require that at least one locale is enabled for the project before it can be saved (regardless if it's localizable or read-only).

Resources:
https://stackoverflow.com/questions/31831620/can-i-make-at-least-one-field-a-requirement-on-a-django-model
https://docs.djangoproject.com/en/dev/ref/forms/validation/#cleaning-and-validating-fields-that-depend-on-each-other

Reply:
Alright will go ahead and implement this